cypress/e2e/project-list.cy.ts

```diff
@@ -14,6 +14,49 @@ describe("Project List", () => {
     });
   };
 
+  const testFetchError = () => {
+    describe("Project list - Error", () => {
+      beforeEach(() => {
+        cy.visit("http://localhost:3000/dashboard");
+      });
+      it("error message displayed on failed request", () => {
+        // intercept request with error
+        cy.intercept("GET", "https://prolog-api.profy.dev/project", {
+          body: {},
+          statusCode: 400,
+        }).as("getProjectError");
+
+        cy.wait(4000);
+
+        cy.get("[data-cy=errorContainer]").should("be.visible");
+      });
+
+      it("data is successfully retrieved and displayed after inital error", () => {
+        // intercept request with error
+        cy.intercept("GET", "https://prolog-api.profy.dev/project", {
+          body: {},
+          statusCode: 400,
+        }).as("getProjectError");
+
+        cy.wait(4000);
```

Comment:
You could try to use the alias of the request to wait for the response similar to line 52.